documentation/modules/mtv-overview-page.adoc

```diff
@@ -10,7 +10,34 @@ The {project-first} *Overview* page displays system-wide information about migra
 
 If you have Administrator privileges, you can access the *Overview* page by clicking *Migration* -> *Overview* in the {ocp} web console.
 
-The *Overview* page displays the following information:
+The *Overview* page has 3 tabs:
+
+* Overview
+* YAML
+* Metrics
+
+[id="overview-tab_{context}"]
+== Overview tab
+
+The *Overview* tab lets you see:
+
+* Operator: The namespace on which the {project-short} Operator and the status of the Operator
```

Comment:
```suggestion
* Operator: The namespace on which the {project-short} Operator is deployed and the status of the Operator
```
or maybe in which the {project-short} runs?

Reply:
@ahadas I went with "is deployed" because it might be deployed but not running (right?).